website/templates/project/view_file.mako

```diff
@@ -6,12 +6,22 @@
 <%def name="title()">${file_name | h}</%def>
 <div class="row">
   <div class="col-sm-5">
-    <h2 class="break-word">
-      ${file_name | h}
-      % if file_revision:
-        <small>&nbsp;${file_revision | h}</small>
-      % endif
-    </h2>
+     <h2 class="break-word">
+       ## Split file name into two parts: with and without extension
+       <%
+        extStart = file_name.rfind(".")
```

Comment:
If the file does not have an extension, but it does have a "." in the title, whatever comes after the "." is treated like an extension and the line breaks:  
![screen shot 2015-10-26 at 4 24 31 pm](https://cloud.githubusercontent.com/assets/6414394/10741546/b5c001fe-7bfe-11e5-9d04-ddf67a4cb9eb.png)  

![screen shot 2015-10-26 at 4 24 41 pm](https://cloud.githubusercontent.com/assets/6414394/10741554/bb06e09c-7bfe-11e5-97ae-a9a6b2378f46.png)


 Is there a more reliable way to get the file extension?